### ISSUE TITLE
[iOS]Display rotation and screen control layout

### ISSUE THREAD

**Issue by surumen41** [hrydgard/ppsspp#7861]
I got a question about on screen control layout as I change display rotation with portrait.
Portrait mode is so comfortable for me with playing my iPad Air, but I couldn't finda way to make screen buttons portrait as well.

how can I do it  with my iPad passpp just like the image which comes from Android ver. ?

v.1.0.1-756-g1ba2b1c

![image](https://cloud.githubusercontent.com/assets/13342858/8689700/3a867838-2ae6-11e5-908b-dc75e7c65d78.jpg)

**Comment by unknownbrackets** [comment]
This is a duplicate of #7777.

But, isn't that a screenshot of the game where characters are supposed to be displaced?  They aren't displaced.  Either I'm wrong (likely, I don't have the game) or this is very interesting.

See #5335.

-[Unknown]

**Comment by surumen41** [comment]
should I think passpp for iOS had attain the feature of changing screen orientation somehow after #7777, but not of screen button layout along with it yet? Because, I succeed in changing rotation with my version.

And, yes this is that game.

**Comment by unknownbrackets** [comment]
We have two rotation options.  One of them is just for rotating the display, the other is for rotating and locking the orientation.  I think you want the latter which isn't implemented yet for iOS.

-[Unknown]
